devtools/projects/ng-devtools/src/lib/devtools-tabs/router-tree/router-tree.component.html

```diff
@@ -176,6 +188,19 @@ <h2 class="router-title">Routes Details</h2>
                   (actionBtnClick)="viewFunctionSource(data.title, 'title')"
                 ></tr>
               }
+
+              @if (data.runGuardsAndResolvers) {
+                <tr
+                  ng-route-details-row
+                  label="RunGuardsAndResolvers"
+                  dataKey="runGuardsAndResolvers"
+                  [data]="data"
+                  [actionBtnType]="hasStaticOptionRunGuardsAndResolvers() ? 'view-source' : 'none'"
+                  [actionBtnTooltip]="hasStaticOptionRunGuardsAndResolvers() ? 'View source' : 'none'"
```

Comment:
```suggestion
                  actionBtnTooltip="View source"
```

No need to have the check. The tooltip is visible only if there is a button anyway.

Reply:
I think it's necessary because if we set `[actionBtnType]="'view-source'"` directly, it will always show like this when that value is a static string.
For now, the only way to avoid that behavior is by using this flag.

<img width="301" height="337" alt="image" src="https://github.com/user-attachments/assets/c81bf5e9-9954-4560-ae66-545bf1743f9c" />